pkgs/development/libraries/libid3tag/default.nix

```diff
@@ -29,6 +20,16 @@ stdenv.mkDerivation rec {
     )
   '';
 
+  strictDeps = true;
+
+  setOutputFlags = false;
+
+  outputs = [ "out" "dev" ];
+
+  nativeBuildInputs = [ gperf_3_0 ];
+
+  propagatedBuildInputs = [ zlib ];
```

Comment:
I don't think that belongs into propageted. minidlna likely only needs zlib in its buildInputs.

Reply:
Done!